sdk/keyvault/keyvault-certificates/src/lro/create/poller.ts

```diff
@@ -0,0 +1,69 @@
+import { delay } from "@azure/core-http";
+import { Poller, PollOperationState } from "@azure/core-lro";
+import {
+  CreateCertificatePollOperation,
+  CreateCertificatePollOperationProperties,
+  makeCreateCertificatePollOperation
+} from "./operation";
+import { CertificateOperation } from "../../core/models";
+import {
+  CertificatePolicy,
+  CertificatesClientInterface,
+  CreateCertificateOptions
+} from "../../certificatesModels";
+
+/**
+ * Class that creates a poller that waits until a certificate finishes being created
+ */
+export class CreateCertificatePoller extends Poller<
+  CreateCertificatePollOperationProperties,
+  CertificateOperation
+> {
+  /**
+   * Defines how much time the poller is going to wait before making a new request to the service.
+   * @memberof CreateCertificatePoller
+   */
+  public intervalInMs: number;
+
+  constructor(
```

Comment:
8 arguments feels like a lot. Why not make this an options bag and unpack them with defaults?